ehsm_kms_service/ehsm_kms_server.env

```diff
@@ -0,0 +1,14 @@
+#ehsm_kms_server
+export EHSM_CONFIG_COUCHDB_USERNAME=admin
+export EHSM_CONFIG_COUCHDB_PASSWORD=password
+export EHSM_CONFIG_COUCHDB_SERVER=10.112.240.122
+export EHSM_CONFIG_COUCHDB_PORT=5984
+export EHSM_CONFIG_COUCHDB_DB=ehsm_kms_db2
```

Comment:
what is this DB used for?  
Both nonce and deny_list in the control server are based-on memory.

Reply:
This file is just for local test run. I will delete it.